thali/thalilogger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var winston = require('winston');
```

Comment:
if we don't specify any transports, the default output is console.   Created the issue #74.